$NetBSD: patch-ac,v 1.2 2008/09/01 06:57:58 dholland Exp $

This fixes an error that will only occur on machines fast enough to
compile through two directories in the "libray" directorie in 1 
second: the upto-date check of libray.a will fail because it was already
touched in this very second. Slowing things down here will help.
 
(I seem to remember some hacking about this before, maybe in make(1),
but I don't remember, and I won't start hacking make(1) now ;-)

(Note from 2008: still pertinent, because the build misuses ar to add
objects into the same .a file repeatedly. I could hack it to not do
that, but for now I'm choosing not to. However, I am fixing the
makefile to stop on error.)

--- libray/Makefile.orig	1992-02-09 22:03:05.000000000 -0500
+++ libray/Makefile	2008-09-01 02:06:03.000000000 -0400
@@ -10,7 +10,7 @@ SHELL = /bin/sh
 
 default:
 	for i in $(STUFF); do \
-		(cd $$i && $(MAKE)); \
+		(echo $$i ; cd $$i && $(MAKE) && sleep 1) || exit 1; \
 	done
 
 #
@@ -20,11 +20,11 @@ default:
 
 depend:
 	for i in $(STUFF); do \
-		(cd $$i && $(MAKE) depend); \
+		(cd $$i && $(MAKE) depend) || exit 1; \
 	done
 
 clean:
 	for i in $(STUFF); do \
-		(cd $$i && $(MAKE) clean); \
+		(cd $$i && $(MAKE) clean) || exit 1; \
 	done
 	rm -f libray.a
